cohorts/load.py

```diff
@@ -537,7 +537,7 @@ def _load_single_patient_neoantigens(self, patient, variant_type,
             if df_isovar is None:
                 import logging
                 logging.disable(logging.INFO)
-                rna_bam_file = AlignmentFile(patient.rna_bam.path)
+                rna_bam_file = AlignmentFile(patient.tumor_sample.bam_path_rna)
```

Comment:
`tumor_sample` could be `None` worth checking? Not sure what to do in that case.